lib/src/core/authentication-core.ts

```diff
@@ -62,82 +62,102 @@ export class AuthenticationCore<T> {
         this._oidcProviderMetaData = async () => await this._dataLayer.getOIDCProviderMetaData();
     }
 
-    public async getAuthorizationURL(config?: AuthorizationURLParams, userID?: string): Promise<string> {
-        const authorizeEndpoint: string = (await this._dataLayer.getOIDCProviderMetaDataParameter(
-            AUTHORIZATION_ENDPOINT as keyof OIDCProviderMetaData
-        )) as string;
-
+    public async getAuthorizationURLParams(
+        config?: AuthorizationURLParams,
+        userID?: string
+    ): Promise<Map<string, string>> {
         const configData: StrictAuthClientConfig = await this._config();
-
-        if (!authorizeEndpoint || authorizeEndpoint.trim().length === 0) {
-            throw new AsgardeoAuthException(
-                "JS-AUTH_CORE-GAU-NF01",
-                "No authorization endpoint found.",
-                "No authorization endpoint was found in the OIDC provider meta data from the well-known endpoint " +
-                "or the authorization endpoint passed to the SDK is empty."
-            );
-        }
-
-        const authorizeRequest: URL = new URL(authorizeEndpoint);
-
-        const authorizeRequestParams: Map<string, string> = new Map<string, string>();
-
+  
+        const authorizeRequestParams: Map<string, string> = new Map<
+        string,
+        string
+      >();
+  
         authorizeRequestParams.set("response_type", "code");
         authorizeRequestParams.set("client_id", configData.clientID);
-
+  
```

Comment:
Remove the additional space characters. Please check and fix the other places as well.

```suggestion

```

Reply:
ack.